components/feed/FeedItem.tsx

```diff
@@ -85,6 +82,13 @@ const FeedItem = ({
               />
             </svg>
           </div>
+          <div>
+            <LikeButton
+              userId={currentUser.id}
+              postId={data.id}
+              likes={data.likes}
```

Comment:
Can we calculate this somehow based on the likesCol instead of storing a separate number? feels like it would be easy to get out of sync